rxpm/src/main/kotlin/me/dmdev/rxpm/widget/InputControl.kt

```diff
@@ -108,5 +108,6 @@ infix fun InputControl.bindTo(editText: EditText) {
         .skipInitialValue()
         .filter { !editing }
         .map { it.toString() }
+        .filter { it != text.value }
```

Comment:
It will be better if combined with the filter above:
`filter { !editing && it != text.value }`

Reply:
Just wanted to avoid another `toString()` before `map` to perform filtering..

Reply:
Maybe something like `.filter { !editing && !text.value.contentEquals(it) }` ?

Reply:
or even `.filter { !editing && text.valueOrNull()?.contentEquals(it) != true }` ?

Reply:
yes that's better

Reply:
done!